hackathon_site/dashboard/frontend/src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -2,12 +2,69 @@ import React from "react";
 import Typography from "@material-ui/core/Typography";
 import Header from "components/general/Header/Header";
 import { hackathonName } from "constants.js";
+import OrderCard from "components/orders/OrderCard/OrderCard";
+import { useDispatch, useSelector } from "react-redux";
+import styles from "./AdminDashboard.module.scss";
+import Grid from "@material-ui/core/Grid";
+import Button from "@material-ui/core/Button";
+import {
+    adminOrderSelectors,
+    getOrdersWithFilters,
+    setFilters,
+} from "../../slices/order/adminOrderSlice";
+import { OrderFilters } from "../../api/types";
 
 const AdminDashboard = () => {
+    const dispatch = useDispatch();
+    const orders = useSelector(adminOrderSelectors.selectAll);
+    const pendingFilter: OrderFilters = {
+        status: ["Submitted"],
```

Comment:
Is this the only status for pending orders? `Ready for Pickup` I believe is also a pending order

Reply:
Ah, yes you're right, I've fixed that as well, thanks